test/test_integration.py

```diff
@@ -26,19 +26,19 @@ def test_start_beebop():
 
     assert docker_util.network_exists("beebop_nw")
     assert docker_util.volume_exists("beebop_storage")
-    assert docker_util.container_exists("beebop_api")
-    assert docker_util.container_exists("beebop_redis")
-    assert docker_util.container_exists("beebop_server")
-    assert docker_util.container_exists("beebop_proxy")
-    assert len(docker_util.containers_matching("beebop_worker_", False)) == 2
+    assert docker_util.container_exists("beebop-api")
```

Comment:
I'm guessing these container names have changed in later versions of constellation..?